modules/relocation-of-openshift-jenkins-images.adoc

```diff
@@ -1,8 +1,13 @@
+// Module included in the following assemblies:
+//
+// * cicd/jenkins/openshift-jenkins.adoc
 :_content-type: CONCEPT
 
 [id="relocation-of-openshift-jenkins-images_{context}"]
 = Relocation of OpenShift Jenkins images
 
+{product-title} 4.11 makes significant changes to the location and availability of specific OpenShift Jenkins images. Additionally, you can configure when and how to update these images.
```

Comment:
I added this abstract to meet the checklist requirement of providing a single-paragraph TLDR.